### PR TITLE
MM-36112 - hide upgrade references when only one product

### PR DESCRIPTION
**Pull request by pablovelezvidal**
#### Summary
This PR adds the logic to make sure to hide any reference to upgrade as long as only one product is handled for the cloud offering.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-36112

#### Screenshots
Captura de pantalla 2021-06-03 a las 21 29 30

#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: components/admin_console/billing/billing_subscriptions/billing_subscriptions.tsx [hunk 1]
```diff
@@ -21,6 +21,7 @@ export const contactSalesCard = (
     trialQuestionsLink: any,
     subscriptionPlan: string | null,
     onUpgradeMattermostCloud: () => void,
+    productsLenght: number,
```
Comment (marianunez) [inline components/admin_console/billing/billing_subscriptions/billing_subscriptions.tsx:24]: typo: 

```suggestion
    productsLength: number,
```

**Review by marianunez — APPROVED** [review]

**Review by stevemudie — APPROVED** [review]
Looks good!

Offer to contact sales about an annual discount displays where the previous message about upgrading was located.

**Review by nickmisasi — APPROVED** [review]

**Comment by marianunez** [comment]
@pablovelezvidal this needs to be cherry-picked to cloud given that the next cloud release does not include multiple products we don't want the upgrade option there yet. 

/cc @amyblais

**Pull request merged**